### PR TITLE
Add GitHub Actions

### PR DESCRIPTION
**Pull request by gracieflores**
Switching from Travis to Github Actions before upgrading Django.

### CODE REVIEW

**Comment by gracieflores** [comment]
@ldko @somexpert This is ready for review.

File: README.md [hunk 1]
```diff
@@ -20,15 +20,15 @@ Requirements
...
+- Python 3.6-3.7
 
 
 Installation
 --------------------------
 
 1.  Install the app
     ```sh
-          $ pip install major-event-log
+          $ pip install django-major-event-log
```
Comment (ldko) [inline README.md]: Do we want to maintain this app on pypi? Or should we change this to a clone and an install from the local copy?

**Review by ldko — COMMENTED** [review]
Looks like this is working and looks good. I just wonder about us changing installation from pypi.

**Review by ldko — APPROVED** [review]
Looks good to me. Thanks, @gracieflores :+1:

File: tox.ini [hunk 2]
```diff
@@ -9,7 +13,8 @@ max-line-length = 99
 deps =
     -rrequirements-test.txt
     django22: Django~=2.2.17
-commands = python manage.py test tests/
+    pytest
```
Comment (gracieflores) [inline tox.ini]: I realized we don't need `pytest` here because it gets installed in the `requirements-test.txt`. So I will remove that.

**Comment by somexpert** [comment]
So I think this looks good and can be merged, but I have one question: is there an easy way to make flake8 _warnings_ more visible? I see that we've got one tiny little whitespace issue, but it doesn't show up anywhere unless you manually look at the details of the runs. It's not a big deal, as we've _told_ it to only fail on certain things, but I was just wondering if you're aware of a way to show some sort of non-failing warnings on runs? It might not exist. In my mind I was just wondering if they had like a yellow indication for the run instead of a green checkmark or red 'x'. But anyways, the changes here do look good to me! :thumbsup:

**Comment by gracieflores** [comment]
@somexpert It looks like they only have three status types (success, cancelled, and failure). We would just have to know to look at the details of the that job. We can always change the job for linting if needed.

**Pull request merged**